.github/workflows/ci.yml

```diff
@@ -16,3 +16,5 @@ jobs:
   plugin-ci:
     uses: mattermost/actions-workflows/.github/workflows/plugin-ci.yml@main
     secrets: inherit
+    with:
+      golang-version: "1.22"
```

Comment:
This change is unique in all the others in that I couldn't get `mattermost-govet` as built to run with `go vet`, it seems because the toolchain version installed was different than `go.mod`.

Aligning this here fixed the issue... but that technically potentially changes the Go version we're building the core plugin with (even though it should have been that way), and I'm surfacing here for visibility.